test/Microsoft.Azure.Devices.Edge.Test/Module.cs

```diff
@@ -17,6 +17,44 @@ public class Module : SasManualProvisioningFixture
         const string SensorName = "tempSensor";
         const string DefaultSensorImage = "mcr.microsoft.com/azureiotedge-simulated-temperature-sensor:1.0";
 
+        [Test]
+        [Category("CentOsSafe")]
+        public async Task CertRenew()
```

Comment:
What does this test do?

Reply:
it sets the renew timestamp at 1 minutes and then it checks that edgeHub was restarted by edgeAgent by checking reported properties restartCount to be 1